docs/getting_started/installation/gpu.rocm.inc.md

```diff
@@ -11,9 +11,9 @@ vLLM supports AMD GPUs with ROCm 6.3 or above, and torch 2.8.0 and above.
 # --8<-- [end:installation]
 # --8<-- [start:requirements]
 
-- GPU: MI200s (gfx90a), MI300 (gfx942), MI350 (gfx950), Radeon RX 7900 series (gfx1100/1101), Radeon RX 9000 series (gfx1200/1201)
-- ROCm 6.3 or above
-    - MI350 requires ROCm 7.0 or above
+- GPU: MI210/MI250 (gfx90a), MI300/MI325 (gfx942), MI350/MI355 (gfx950), Radeon RX 7900 series (gfx1100/1101), Radeon RX 9000 series (gfx1200/1201)
+- ROCm 6.0 or above
```

Comment:
I think we support either 5.7 or 6.0 onwards per the comment in the docker build steps; updated to clarify